src/cljs/bluegenes/components/loader.cljs

```diff
@@ -20,3 +21,16 @@
      [:div.loader-organism.dot]
      [:div.loader-organism.three]]
     [:div.loader-organism.four]]])
+
+(defn mine-loader []
+  (let [show? @(subscribe [:show-mine-loader?])
+        {:keys [logo name]} @(subscribe [:current-mine])]
+    (when show?
+      [:div.mine-loader
+       [:div.mine-loader-dialog
+        (if (not-empty logo)
+          [:img {:src logo}]
+          [:svg.icon-2x.icon-intermine [:use {:xlinkHref "#icon-intermine"}]])
+        [loader name]
+        [:p "It's taking an unusual amount of time to load this mine."]
+        [:p "Consider contacting the maintainers or switching to a different mine."]]])))
```

Comment:
would it be a big pain to add the maintainer info from the registry if known?

Reply:
- happy to have that be a ticket for the future if it's easier

Reply:
I didn't look into it, since I feel it's very likely that any maintainer email present on the registry is not observed or prepared to handle such enquiries. It's also possible that the problem lies with the user's connection, or the maintainers of the bluegenes instance (ie. not just the mine you're connecting to).
If you still think you want this then you can go ahead and open a ticket. [=

Reply:
seems reasonable!